netket/graph/lattice.py

```diff
@@ -78,8 +81,9 @@ def create_points(basis_vectors, extent, atom_coords, pbc):
 
 def get_true_edges(basis_vectors, atoms, cellANDlabel_to_site, extent):
     atoms_positions = dicts_to_array(atoms, "r_coord")
+    tol = 1e-5
     naive_edges = get_edges(
-        atoms_positions, _np.linalg.norm(basis_vectors, axis=1).max()
+        atoms_positions, _np.linalg.norm(basis_vectors, axis=1).max() + tol
```

Comment:
why you add the tolerance here?

Reply:
Yeah, I'll have to check if the kdtree allows for this tolerance as an argument. If not I'll just add it inside get_edges as suggested.